llvm/lib/Target/X86/X86SelectionDAGInfo.cpp

```diff
@@ -86,7 +86,7 @@ SDValue X86SelectionDAGInfo::EmitTargetCodeForMemset(
       ValReg = X86::EAX;
       Val = (Val << 8)  | Val;
       Val = (Val << 16) | Val;
-      if (Subtarget.is64Bit() && Alignment > Align(8)) { // QWORD aligned
+      if (Subtarget.is64Bit() && Alignment > Align(4)) { // QWORD aligned
```

Comment:
Maybe use `>= Align(8)` for better understanding?

Reply:
I'm trying to mimic the style of the function I'm modifying (which does > Align(2) but I guess that also works.